Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -10,7 +10,12 @@ public class GraphTraverser: GraphTraversing {
     public var path: AbsolutePath { graph.path }
     public var workspace: Workspace { graph.workspace }
     public var projects: [AbsolutePath: Project] { graph.projects }
-    public var targets: [AbsolutePath: [String: Target]] { graph.targets }
+    public var targets: [AbsolutePath: [String: Target]] {
```

Comment:
Should we consider making this a method since we iterate over projects and their targets? Computed property should imho be reserved for O(1) operations

Reply:
I updated it!